lib/cube_browser/explorer.py

```diff
@@ -258,9 +260,8 @@ def _goplot(self, sender):
                 x_name = cube.coord(axis='X').name()
                 y_name = cube.coord(axis='Y').name()
                 if x_name == pc_x_name and y_name == pc_y_name:
-                    projection = cube.coord_system().as_cartopy_projection()
-                    ax = fig.add_subplot(sub_plots + spl,
-                                         projection=projection)
+                    proj = iplt.default_projection(cube) or ccrs.PlateCarree()
```

Comment:
I can see Mark saying "keep it clean", so I would say that is not an option at this point.